tests/standalone/BotocoreUsing.py

```diff
@@ -0,0 +1,18 @@
+import botocore.session
```

Comment:
Is botocore really woth being tested separately, wouldn't boto3 use that aleady?

This also seems not mocked, maybe you mean to remove it.